core/linux-dist/x11.cpp

```diff
@@ -72,6 +76,16 @@ void input_x11_handle()
 					{
 						die("death by escape key");
 					}
+#if FEAT_HAS_NIXPROF
+					else if (e.type == KeyRelease && e.xkey.keycode == 76) // F10 button
+					{
+						if (sample_Switch(97)) {
```

Comment:
97 hz is wayy too low to get accurate profiling results. I originally used 1K to 10K